tests/foreman/api/test_computeresource_libvirt.py

```diff
@@ -20,132 +20,115 @@
 
 :Upstream: No
 """
-from random import randint
-
 import pytest
 from fauxfactory import gen_string
-from nailgun import entities
 from requests.exceptions import HTTPError
 
 from robottelo.config import settings
+from robottelo.constants import FOREMAN_PROVIDERS
 from robottelo.constants import LIBVIRT_RESOURCE_URL
 from robottelo.utils.datafactory import invalid_values_list
 from robottelo.utils.datafactory import parametrized
 from robottelo.utils.datafactory import valid_data_list
 
-pytestmark = [
-    pytest.mark.skip_if_not_set('libvirt'),
-]
-
+pytestmark = [pytest.mark.skip_if_not_set('libvirt')]
 
-@pytest.fixture(scope="module")
-def setup():
-    """Set up organization and location for tests."""
-    setupEntities = type("", (), {})()
-    setupEntities.org = entities.Organization().create()
-    setupEntities.loc = entities.Location(organization=[setupEntities.org]).create()
-    setupEntities.current_libvirt_url = LIBVIRT_RESOURCE_URL % settings.libvirt.libvirt_hostname
-    return setupEntities
+LIBVIRT_URL = LIBVIRT_RESOURCE_URL % settings.libvirt.libvirt_hostname
 
 
-@pytest.mark.tier1
-@pytest.mark.parametrize('name', **parametrized(valid_data_list()))
-def test_positive_create_with_name(setup, name):
-    """Create compute resources with different names
+@pytest.mark.e2e
+def test_positive_crud_libvirt_cr(module_target_sat, module_org, module_location):
+    """CRUD compute resource libvirt
 
-    :id: 1e545c56-2f53-44c1-a17e-38c83f8fe0c1
+    :id: 1e545c56-2f53-44c1-a17e-38c83f8fe0c2
 
     :expectedresults: Compute resources are created with expected names
 
     :CaseImportance: Critical
 
     :CaseLevel: Component
-
-    :parametrized: yes
     """
-    compresource = entities.LibvirtComputeResource(
-        location=[setup.loc],
+    name = gen_string('alphanumeric')
+    description = gen_string('alphanumeric')
+    display_type = 'spice'
+    libvirt_url = 'qemu+tcp://libvirt.example.com:16509/system'
```

Comment:
Can we make it a constant `libvirt_url`  ?

Reply:
its just a example url, we already have a constant for actual URL